gauth.go

```diff
@@ -42,22 +47,45 @@ func main() {
 
 	currentTS, progress := gauth.IndexNow()
 
-	tw := tabwriter.NewWriter(os.Stdout, 0, 8, 1, ' ', 0)
-	fmt.Fprintln(tw, "\tprev\tcurr\tnext")
-	for _, record := range cfg {
-		name, secret := record[0], record[1]
-		prev, curr, next, err := gauth.Codes(secret, currentTS)
-		if err != nil {
-			log.Fatalf("Generating codes: %v", err)
+	if *useCSV {
+		cw := csv.NewWriter(os.Stdout)
+		for _, record := range cfg {
+			name, secret := record[0], record[1]
+			_, curr, next, err := gauth.Codes(secret, currentTS)
```

Comment:
Is there some reason to omit the previous code in the CSV output? If the goal is to make the output of the tool machine-readable, I'd suggest it should contain the same information as the human-readable version.

Reply:
Included.